nixos/modules/tasks/network-interfaces.nix

```diff
@@ -1039,6 +1047,20 @@ in
       '';
     };
 
+    networking.ipv6PrivacyExtensions = mkOption {
+      default = if cfg.enableIPv6 then "default" else "disabled";
+      type = types.enum (lib.attrNames tempaddrValues);
+      description = ''
+        Whether to enable IPv6 Privacy Extensions for interfaces not
+        specified explicitly in <option>networking.interfaces</option>.
+
+        This sets the ipv6.conf.*.use_tempaddr sysctl for all
```

Comment:
This should probably mention `networking.interfaces.<name>.tempAddress` too, or maybe just that.

Reply:
I thought the reference to `networking.interfaces` would be enough. I have added a reference in the other direction though, since I think the global option is likely to be more useful in typical scenarios.

Reply:
Ah, I think missed that. Still, how about `Whether to enable IPv6 Privacy Extensions for interfaces not specified explicitly with <option>networking.interfaces.&lt;name&gt;.tempAddress.</option>`?